### DISCUSSION TITLE
Feature: Mention of Contributors

### DISCUSSION THREAD

**Discussion by CBID2** [open-sauced/docs#335] [Ideas]
### Type of feature

🍕 Feature

### Current behavior

I was looking through the repo's readme file and noticed that there is no mention of the contributors. It can cause them to feel disheartened. 

### Suggested solution

My colleague Josh Goldberg has created a [GitHub Action](https://github.com/marketplace/actions/all-contributors-auto-action) that automatically posts contributors if they raise an issue or get their PR merged. I've seen it in action(no pun intended) and it's pretty effective productivity-wise.  

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

**Comment by bdougie** [comment]
We have some work we are doing to add all contributors to all opensauced projects, which is what I believe Josh is using. 

Should be available in the next few weeks and then we can organize how we display contributors.

**Comment by CBID2** [comment]
> We have some work we are doing to add all contributors to all opensauced projects, which is what I believe Josh is using. 
> 
> Should be available in the next few weeks and then we can organize how we display contributors.

Great @bdougie! I highly recommend highlighting non-code contributions to make open source contributors, especially those who are new in coding feel welcomed! :)
